app/frontend/wallet/shelley/shelley-js-crypto-provider.ts

```diff
@@ -129,14 +130,18 @@ const ShelleyJsCryptoProvider = ({
     return ShelleySignedTransactionStructured(txAux, witnesses, meta)
   }
 
-  function isFeatureSupported(feature) {
+  function isFeatureSupported(feature: CryptoProviderFeatures) {
     return true
   }
 
-  function ensureFeatureIsSupported(feature) {
+  function ensureFeatureIsSupported(feature: CryptoProviderFeatures) {
```

Comment:
`CryptoProviderFeatures` looks like a collection of something at first sight and it looks odd for `feature` to be of such type, so I'd consider renaming the type to `CryptoProviderFeatureEnum` for the sake of clarity

Reply:
I ll refactor the enum names, thanks :+1: